src/main/kotlin/com/team4099/robot2022/subsystems/SwerveModule.kt

```diff
@@ -147,37 +147,54 @@ class SwerveModule(
     Logger.addSource("$label Drivetrain", "Drive Position") { driveSensor.position.inFeet }
     Logger.addSource("$label Drivetrain", "Steering Position") { steeringPosition.inDegrees }
 
+    Logger.addSource("$label Drivetrain", "Steering Position Raw") { potentiometer.get() }
+
     Logger.addSource(
         "Drivetrain Tuning",
         "$label Azimuth kP",
-        { Constants.Drivetrain.PID.STEERING_KP },
+        { DrivetrainConstants.PID.STEERING_KP },
         { newP -> steeringFalcon.config_kP(0, newP) },
         false)
 
-    steeringConfiguration.slot0.kP = Constants.Drivetrain.PID.STEERING_KP
-    steeringConfiguration.slot0.kI = Constants.Drivetrain.PID.STEERING_KI
-    steeringConfiguration.slot0.kD = Constants.Drivetrain.PID.STEERING_KD
-    steeringConfiguration.slot0.kF = Constants.Drivetrain.PID.STEERING_KFF
+    steeringConfiguration.slot0.kP = DrivetrainConstants.PID.STEERING_KP
+    steeringConfiguration.slot0.kI = DrivetrainConstants.PID.STEERING_KI
+    steeringConfiguration.slot0.kD = DrivetrainConstants.PID.STEERING_KD
+    steeringConfiguration.slot0.kF = DrivetrainConstants.PID.STEERING_KFF
+    //    steeringConfiguration.motionCruiseVelocity =
+    //        steeringSensor.velocityToRawUnits(DrivetrainConstants.STEERING_VEL_MAX)
+    //    steeringConfiguration.motionAcceleration =
+    //        steeringSensor.accelerationToRawUnits(DrivetrainConstants.STEERING_ACCEL_MAX)
     steeringConfiguration.motionCruiseVelocity =
```

Comment:
ok the commented lines above this do literally exactly what you're doing here lol avoid code replication by using the steering sensor thing